src/main/java/org/springframework/data/elasticsearch/repository/query/ElasticsearchStringQuery.java

```diff
@@ -15,18 +15,30 @@
  */
 package org.springframework.data.elasticsearch.repository.query;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.elasticsearch.search.aggregations.*;
+import org.elasticsearch.search.aggregations.bucket.terms.TermsAggregationBuilder;
+import org.elasticsearch.search.builder.SearchSourceBuilder;
```

Comment:
no Elasticsearch classes here. This will not work with the next Elasticsearch client.

Reply:
can you point me to the new client branch you're talking about? I need to review it to have a better context of this problem